core-tests/src/test/java/org/mule/runtime/core/el/mvel/MVELMapHandlingTestCase.java

```diff
@@ -82,7 +83,7 @@ private void assertMapKey(Event event, String key, Object expectedValue) throws
   }
 
   private void runExpressionAndExpect(String expression, Object expectedValue, Event event) throws Exception {
-    Object result = el.evaluate(expression, event, getTestFlow(muleContext)).getValue();
+    Object result = el.evaluate(expression, event, fromSingleComponent("flowName")).getValue();
```

Comment:
Use a constant, there should already be flow name constants in Abstract test cases.